app/services/flow/flows/provider_capital.rb

```diff
@@ -72,7 +72,9 @@ class ProviderCapital < FlowSteps
         },
         income_summary: {
           path: ->(application) { urls.providers_legal_aid_application_income_summary_index_path(application) },
-          forward: :outgoings_summary
+          forward: ->(application) do
+            application.checking_answers? ? :means_summaries : :outgoings_summary
+          end
```

Comment:
instead of changing the `forward` key, you can add the `check_answers` key:
```ruby
  check_answers: :means_summaries
```
The logic you are adding is already supported by the flow service:
if the application is in a "check answers" state and a `check_answers` key is defined, then use that.
Otherwise, use the `forward` key.
(https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/master/app/services/flow/base_flow_service.rb#L29)

Reply:
awesome I didn't know that